locales/fr/messages.po

```diff
@@ -14,273 +14,276 @@ msgstr ""
 
 #: lib/actionButtons.js:186
 msgid "This is your current task."
-msgstr ""
+msgstr "Voici votre tâche actuelle."
 
 #: lib/actionButtons.js:144
 msgid "This is your first task."
-msgstr ""
+msgstr "Voici votre première tâche."
 
 #: lib/actionButtons.js:73
 #: lib/actionButtons.js:180
 msgid "This workflow is now complete."
-msgstr ""
+msgstr "Ce flux de travail est maintenant terminé."
 
 #: pages/contact_us.js:35
 msgid "To report bugs or problems or to provide feedback about the Navigator software, please contact <0>TechnicalSupportEmailAddress</0>. If relevant capture a screenshot of the error you are receiving or the task in question."
-msgstr ""
+msgstr "Pour signaler des bogues ou des problèmes ou fournir des commentaires sur le logiciel Navigateur, veuillez contacter <0>TechnicalSupportEmailAddress</0>. Si pertinent, effectuez une capture d'écran de l'erreur que vous recevez ou de la tâche en question."
 
 #: pages/contact_us.js:27
 msgid "UNAIDS and its partners are committed to providing the highest level of support to country teams using its platforms and models during the estimates process. You can contact us in one of two ways:"
-msgstr ""
+msgstr "L'ONUSIDA et ses partenaires s'engagent à fournir le plus haut niveau de soutien aux équipes nationales utilisant ses plateformes et modèles durant le processus des estimations. Vous pouvez nous contacter via l'un des deux moyens:"
 
 #: components/ErrorComponents.js:29
 #: components/ErrorPagePopup.js:40
 msgid "View Error Logs"
-msgstr ""
+msgstr "Consulter les Journaux d'Erreurs"
 
 #: pages/login.js:45
 msgid "Welcome to the UNAIDS HIV Estimates Navigator. Your HIV estimates journey begins here!"
-msgstr ""
+msgstr "Bienvenue dans le Navigateur des Estimations VIH de l'ONUSIDA. Votre voyage vers les estimations VIH commence ici!"
 
 #: components/ActionButtons.js:121
 #: components/Sidebar.js:8
 msgid "What's Next?"
-msgstr ""
+msgstr "Quelle est la Prochaine Étape?"
 
 #: pages/index.js:355
 msgid "Workflow: {workflowId}"
-msgstr ""
+msgstr "Flux de travail: {workflowId}"
 
 #: lib/actionButtons.js:16
 msgid "You are currently previewing a future task."
-msgstr ""
+msgstr "Vous êtes actuellement en train de prévisualiser une tâche future."
 
 #: pages/index.js:292
 msgid "You are previewing a future task"
-msgstr ""
+msgstr "Vous êtes en train de prévisualiser une tâche future"
 
 #: pages/no_datasets.js:13
 msgid "Your ADR account does not have access to any Country HIV Estimates Datasets. Please make sure you are logged into the correct account and you are added as an editor member of the country organisation or a dataset collaborator."
-msgstr ""
+msgstr "Votre compte ADR n'a accès à aucun Ensemble de données d'Estimations VIH Pays. S'il vous plaît assurez-vous que vous êtes connecté avec le bon compte et que vous avez êtes ajouté|e comme membre éditeur de l'organisation pays ou comme collaborateur|trice d'un ensemble de données."
 
 #: pages/user_not_found_in_adr.js:13
```

Comment:
It is right like this?

Reply:
Yep, looks good to me. Want to convert to a full PR and I will approve?